src/app/sharedUI/screens/Artwork/Artwork.tsx

```diff
@@ -31,8 +32,17 @@ type RenderArtworkProps = {
 
 const RenderArtwork: React.FC<RenderArtworkProps> = ({ slug }) => {
   const insets = useSafeAreaInsets()
-  const navigation = useNavigation()
+  const navigation = useNavigation<NavigationProp<HomeTabsScreens>>()
   const artworkData = useLazyLoadQuery<ArtworkQuery>(artworkQuery, { slug })
+  const albums = GlobalStore.useAppState((state) => state.albums.albums)
+
+  let numberOfAlbumsTheArtworkAvailable = 0
+
+  albums.map((album) => {
+    if (album.artworkIds.includes(artworkData.artwork?.internalID!)) {
+      numberOfAlbumsTheArtworkAvailable++
+    }
+  })
```

Comment:
what do you think about this?
```suggestion
  const numberOfAlbumsTheArtworkAvailable = useMemo(() => {
    let num = 0
    albums.map((album) => {
      if (album.artworkIds.includes(artworkData.artwork?.internalID!)) {
        num++
      }
    })
    return num
  }, [albums])
```

or even this?
```suggestion
  const numberOfAlbumsTheArtworkAvailable = useMemo(() =>
    albums.filter((album) =>
      album.artworkIds.includes(artworkData.artwork?.internalID!)
    ).length
  }, [albums])
```

Reply:
The first works well. I will use that.